cluster/addons/dashboard/dashboard-controller.yaml

```diff
@@ -1,27 +1,29 @@
+{% set version = "0.1.0-alpha1" -%}
```

Comment:
Where can one see the actual list of versions? The upstream repository doesn't seem to have any official GH releases or tags.